esp-box/esp-box.c

```diff
@@ -69,20 +78,6 @@ void bsp_audio_init(const i2s_config_t *i2s_config)
     ESP_ERROR_CHECK(gpio_config(&io_conf));
 }
 
-void bsp_audio_deinit(void)
```

Comment:
it would have to be `void bsp_audio_deinit(i2s_chan_handle_t handle);` . 

And user would have to call it twice (tx, rx path) to fully deinit the audio. I find calling one deinit function twice confusing and I couldn't think of a better way, so I deleted the audio_deinit function. Plus, I'm not a big fan of single-line wrappers.

Anyway, I'm happy to change it if we can find a better solution.